core/capabilities/ccip/ccipevm/msghasher_test.go

```diff
@@ -328,11 +333,105 @@ func TestMessagerHasher_againstRmnSharedVector(t *testing.T) {
+
+		var (
+			// header fields
+			messageID           = mustBytes32FromString(t, "0xcdad95e113e35cf691295c1f42455d41062ba9a1b96a6280c1a5a678ef801721")
+			sourceChainSelector = cciptypes.ChainSelector(124615329519749607)   // solana
+			destChainSelector   = cciptypes.ChainSelector(16015286601757825753) // sepolia
+			sequenceNumber      = cciptypes.SeqNum(386)
+			nonce               = uint64(1)
+			// message fields
+			// sender is parsed unpadded since its emitted unpadded from EVM.
+			senderAddress = cciptypes.UnknownAddress(hexutil.MustDecode("0x269895AC2a2eC6e1Df37F68AcfbBDa53e62b71B1"))
+			// onRampAddress is parsed padded because its set as a padded address in the offRamp
+			onRampAddress = key.Bytes()
+			dataField     = "0x"
+			// receiver address is parsed padded because its emitted as padded from EVM.
+			receiverAddress = cciptypes.UnknownAddress(hexutil.MustDecode("0x000000000000000000000000269895ac2a2ec6e1df37f68acfbbda53e62b71b1"))
+			// feeToken is parsed unpadded since its emitted unpadded from EVM.
+			// however, it isn't used in the hash. its just set for completion.
+			feeToken       = common.HexToAddress("0x097D90c9d3E0B50Ca60e1ae45F6A81010f9FB534")
+			feeTokenAmount = big.NewInt(114310554250104)
+			feeValueJuels  = big.NewInt(16499514422603741)
+			tokenAmounts   = []cciptypes.RampTokenAmount{
+				{
+					// parsed unpadded since its emitted unpadded from SVM.
+					SourcePoolAddress: cciptypes.UnknownAddress(key.Bytes()),
+					// parsed padded because its emitted padded from EVM.
+					DestTokenAddress: cciptypes.UnknownAddress(hexutil.MustDecode("0x000000000000000000000000b8d6a6a41d5dd732aec3c438e91523b7613b963b")),
+					// extra data always abi-encoded
+					ExtraData: cciptypes.Bytes(hexutil.MustDecode("0x0000000000000000000000000000000000000000000000000000000000000012")),
+					Amount:    cciptypes.NewBigInt(big.NewInt(100000000000000000)),
+					// dest exec data always abi-encoded
+					DestExecData: destExecData,
+				},
+			}
+
+			msg = cciptypes.Message{
+				Header: cciptypes.RampMessageHeader{
+					MessageID:           messageID,
+					SourceChainSelector: sourceChainSelector,
+					DestChainSelector:   destChainSelector,
+					SequenceNumber:      sequenceNumber,
+					Nonce:               nonce,
+					MsgHash:             cciptypes.Bytes32{},
+					OnRamp:              onRampAddress,
+				},
+				Sender:         senderAddress,
+				Data:           hexutil.MustDecode(dataField),
+				Receiver:       receiverAddress,
+				ExtraArgs:      append(evmExtraArgsV2Tag, extraArgsbuf.Bytes()...),
+				FeeToken:       feeToken.Bytes(),
+				FeeTokenAmount: cciptypes.NewBigInt(feeTokenAmount),
+				FeeValueJuels:  cciptypes.NewBigInt(feeValueJuels),
+				TokenAmounts:   tokenAmounts,
+			}
+
+			any2EVMMessage = ccipMsgToAny2EVMMessage(t, msg, sourceChainSelector)
+		)
+
+		const (
+			rmnMsgHash = "0x77c906425ff617174b78bda23625f01d408c039dcd450db2bd1a68f3ae0b5f6c"
```

Comment:
Is this a real RMN hash or are you just putting it here to match the other test case?